starter-code/server.js

```diff
@@ -146,7 +171,11 @@ app.listen(PORT, function() {
 ////////////////////////////////////////
 function loadArticles() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
+
   // Put your response here...
+
+  // There is a query being made so it corresponds to the number 3. This is where I'm a bit confused Carlo, because I see the lines of code below and think that, because of the SELECT and the INSERT, this function corresponds with both READ and CREATE from CRUD. I see a query that's attempting to READ from the table of articles, and if there's nothing there to then CREATE articles by INSERTING the data from hackerIpsum.json into the articles table. I know you said this only really has CREATE from CRUD and I don't quite understand why. I can see how it may lean further towards being CREATE than READ. I'm also not sure what method from article.js this code relates to. Because it CREATES information by INSERTING into the articles table, I could see this pertaining to Article.insertRecord more than any of the other methods.
+
```

Comment:
This is kind of a trick question because its not technically CRUD in terms of CRUD routes. loadArticles is looking for articles to load, if articles already exist in the database it does nothing so its not performing a POST,PUT,GET or DELETE. but if there is nothing in the database then the if block pulls the info from the json file and POST them to the database so in that sense you can say its CREATE